### ISSUE TITLE
Multiple choice input

### ISSUE THREAD

**Issue by mekane** [cronvel/terminal-kit#92]
I love the library, it's really handy and full-featured. The app I'm currently writing would really benefit from the ability to select multiple items. I'm imagining checkbox-like behavior. Basically a single-column menu with a "selected" state. I suppose it would need some way to indicate that the user is done selecting.

e.g.:

Choose an option:  
[  ] Option 1
[x] Option 2  
[  ] Option 3  
[x] Option 4  
Done

**Comment by cronvel** [comment]
@mekane I think it could be an improvement of the [.singleColumnMenu()](https://github.com/cronvel/terminal-kit/blob/master/doc/high-level.md#ref.singleColumnMenu).

**Comment by ASnow** [comment]
I can't find documentation for `SelectListMulti` but think it can help.
I wrote own select for my purpose. It wasn't difficult.

**Comment by cronvel** [comment]
@ASnow Yes that feature is not documented yet, it's part of the *"document model"* I'm still developing in my spare time. 
It's probably time to document it, since it should be stable enough by now, and it's a huge plus for the lib. 
But the documentation itself takes a lot of time! ^^

@mekane If you need that right now, you can browse `sample/document/` and play with `sample/document/select-list-multi-test.js`. A document is a sort of page with widgets.

**Comment by mekane** [comment]
Thanks for the heads up! I'm making do without it for my project, so I'm in no rush. But I appreciate the information.
